### PR TITLE
Coachmark:  RTL pulse beacon animation fix

### LINKED ISSUE #10314: Coachmark animation in RTL does odd things
### Please provide a reproduction of the bug in a codepen:

https://codesandbox.io/s/charming-bird-4wxid

#### Actual behavior:

RTL-Coachmark

#### Expected behavior:

Animation stays over the beacon.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

### PR DESCRIPTION
**Pull request by maxwellred**
#### Pull request checklist

- [] Addresses an existing issue: Fixes #10314 
- [x] Include a change request file using `$ yarn change`

#### Description of changes

Fixes issue where the pulse beacon goes flying to the left in RTL mode

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10381)

### CODE REVIEW

**Comment by maxwellred** [comment]
Hey @leddie24 I was hoping to have you take a quick look at this PR when you have a moment, thanks! It just adds RTL to the Coachmark pulse beacon is all.

File: packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.styles.ts [hunk 1]
```diff
@@ -165,14 +166,20 @@ export function getStyles(props: ICoachmarkStyleProps): ICoachmarkStyles {
...
         top: '50%',
-        left: '50%',
         transform: 'translate(-50%, -50%)',
         width: '0px',
         height: '0px',
         borderRadius: '225px',
         borderStyle: 'solid',
         opacity: '0'
       },
+      getRTL()
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.styles.ts]: hmm; this should have translated automatically

Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.styles.ts]: is it the transform that needs to be flipped? right and left do flip but transforms aren't currently.

Comment (maxwellred) [reply packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.styles.ts]: You are correct. After playing around with it I was able to get the correct transform for RTL. I also noticed that the beak for "Right bottom/center/top Edge" was not visible so I changed that as well

Screen Shot 2019-09-16 at 12 25 51 PM

File: packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ export class Beak extends BaseComponent<IBeakProps, {}> {
         pointOne = `0, 0`;
         pointTwo = `${BEAK_HEIGHT}, ${BEAK_HEIGHT}`;
         pointThree = `0, ${BEAK_WIDTH}`;
-        transform = 'translateX(100%)';
+        transform = getRTL() ? 'translateX(260%)' : 'translateX(100%)';
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx]: 260 is a magic number. It seems a little wrong; shouldn't this be a symmetrical positive/negative 100%? I don't completely undersatnd what's going on here but 260 seems wrong.

Comment (maxwellred) [reply packages/office-ui-fabric-react/src/components/Coachmark/Beak/Beak.tsx]: Found the issue. So we need to RTL-ify the absolute positioning (left/right values) in Coachmark.base.tsx in the _setBeakPosition method

**Pull request merged**